src/lib/PacbioInstrumentTypes.js

```diff
@@ -0,0 +1,79 @@
+import { LabwareTypes } from '@/lib/LabwareTypes'
+
+/**
+ * A collection of methods to deal with Pacbio Instrument Types
+ * we could possibly add these methods to the object itself
+ * which would be better for naming and usage
+ * @module PacbioInstrumentTypes
+ */
+
+/**
+ * @typedef PacbioInstrumentType
+ * @property {String} name
+ * @property {Number} plateCount
+ * @property {LabwareType} labwareType
+ * @property {String} plateClasses
+ * @property {Number} sequencingKitBoxBarcodeLength
+ *
+ */
+const PacbioInstrumentTypes = {
+  Revio: {
+    name: 'Revio',
+    plateCount: 2,
+    labwareType: LabwareTypes.Plate4,
+    plateClasses: 'w-32 mx-auto',
+    sequencingKitBoxBarcodeLength: 28,
+  },
+  SequelIIe: {
+    name: 'Sequel IIe',
+    plateCount: 1,
+    labwareType: LabwareTypes.Plate96,
+    plateClasses: 'w-full',
+    sequencingKitBoxBarcodeLength: 21,
+  },
+}
+
+/**
+ *
+ * @param {String} name
+ * @param {Object} instrumentTypeList
+ * @returns PacbioInstrumentType || undefined
+ * finds the Pacbio Instrument Type by name
+ */
+const findInstrumentByName = (name, instrumentTypeList) => {
```

Comment:
It is. It would be better to do it the way you suggested but we would need to change the component to use the key. from the drop down. I can have a look.